src/main/resources/global/90-init-alfresco.sh

```diff
@@ -26,9 +26,23 @@ fi
 if [ -n "$CATALINA_HOME" ]; then
 
   user="tomcat"
-  if [[ $(stat -c %U /opt/alfresco/alf_data) != "$user" ]]; then
-    chown -R $user:$user /opt/alfresco/alf_data
+  # Check existence of alf_data/contentstore
+  # -> if dir exists, check if the permissions are incorrect
+  # --> if permissions are incorrect, exit with error code
+  # --> else if correct, do nothing
+  # -> else if not exists, chown only alf_data
+  if [[ -e /opt/alfresco/alf_data/contentstore && -e /opt/alfresco/alf_data/contentstore.deleted ]]; then
+    if [[ ($(stat -c %U /opt/alfresco/alf_data/contentstore) != "$user" && $(stat -c %a /opt/alfresco/alf_data/contentstore) -lt 766)\
+     && ($(stat -c %U /opt/alfresco/alf_data/contentstore.deleted) != "$user" && $(stat -c %a /opt/alfresco/alf_data/contentstore.deleted) -lt 766) ]]; then
```

Comment:
Shouldn't this be an OR? When one of contentstore and contentstore.deleted has wrong permissions, we should exit.